docs/jade-getting-started.md

```diff
@@ -366,6 +366,13 @@ export AZURE_SYNAPSE_INITIALIZE=false
 # Pact contract test settings
 export PACT_BROKER_USERNAME=$(cat /tmp/pact-ro-username.key)
 export PACT_BROKER_PASSWORD=$(cat /tmp/pact-ro-password.key)
+
+# Azure B2C authentication settings
+export OIDC_ADDCLIENTIDTOSCOPE=true
+export OIDC_AUTHORITYENDPOINT="https://oauth-proxy.dsp-eng-tools.broadinstitute.org/b2c"
+export OIDC_CLIENTID=bbd07d43-01cb-4b69-8fd0-5746d9a5c9fe
+export OIDC_EXTRAAUTHPARAMS="prompt=login"
+export OIDC_PROFILEPARAM=b2c_1a_signup_signin_tdr_dev
```

Comment:
Could we just set these by default in application.properties instead? 
In trying to simplify our dev setup, I think it's ideal if we can automatically set variables for devs to the right defaults.

Reply:
+1, this would be preferred.